CHANGELOG

```diff
@@ -14,7 +14,7 @@
 * Deepstack: Add Deepstack Gateway [khoinguyendeepstack] #4830
 * Braintree: Additional tests for credit transactions [jcreiff] #4848
 * Rapyd: Change nesting of description, statement_descriptor, complete_payment_url, and error_payment_url [jcreiff] #4849
-
+* IPG: Refactor Credentials [kylene-spreedly] #4854
```

Comment:
Nitpick, but this is not technically a refactor since it changes functionality

Reply:
I updated this too. Lemme know if that makes more sense!